### DISCUSSION TITLE
Enable Killswitch

### DISCUSSION THREAD

**Discussion by RoiBurgonde** [bubuntux/nordlynx#35] [Q&A] [answered]
Hello,

Newbie question:
I’m using Nordvpn, and i wonder if it is possible to enable KILLSWITCH in the docker-compose ?

Thanx in advance for help!

**Comment by RoiBurgonde** [comment]
No idea if it’s possible ?

**Comment by bubuntux** [comment, marked as answer]
Killswitch is enabled by default in this container

**Reply by RoiBurgonde** [reply]
Cool, thanx!!

**Comment by xXDasGoGXx** [comment]
Hi! I hope not to sound redundant.... Is killswitch automatic on https://github.com/bubuntux/nordlynx ?
I am so literal when people say NordVPN, I wanna be sure that is also means this, Nordlynx from that link.
Thank you

**Reply by bubuntux** [reply]
yes, killswitch is enable using iptables
